common-travel.lic

```diff
@@ -72,8 +72,9 @@ module DRCT
     room_num == Room.current.id
   end
 
-  def retreat
+  def retreat(ignored_npcs=Array.new)
     return if DRRoom.npcs.empty?
+    return if DRRoom.npcs.to_set.subtract(ignored_npcs).empty?
```

Comment:
You don't need `to_set`, you could do something like `(DRRoom.npcs - @ignored_npcs).empty?`

This might not be a bad helper to commonize either... it's used in `combat-trainer` and `oshu-manor` (maybe others)

Reply:
Well if there are multiple `owls` or `town guards` in the room or something - to set will dedupe it

Reply:
Ohh TIL i learned
```
>;en echo DRRoom.npcs                                                  
--- Lich: exec1 active.                                                
[exec1: ["falcon", "eagle", "hawk", "kestrel", "eagle", "merlin"]]     
--- Lich: exec1 has exited.                                            
>;en echo DRRoom.npcs - ['eagle']                                      
--- Lich: exec1 active.                                                
[exec1: ["falcon", "hawk", "kestrel", "merlin"]]                       
--- Lich: exec1 has exited.                                            
```